src/backend/distributed/master/citus_tools.sql

```diff
@@ -0,0 +1,180 @@
+/*
+ * citus_tools.sql
+ * Contains definitions of citus_tools UDFs
+ * - citus_run_on_all_workers
+ * - citus_run_on_all_placements
+ * - citus_run_on_all_colocated_placements
+ * - citus_run_on_all_shards
+ *
+ * These functions depends on presense of UDF master_run_on_worker
+ */
+
+CREATE OR REPLACE FUNCTION master_run_on_worker(worker_name text[], port integer[],
+												command text[], parallel boolean,
+												OUT node_name text, OUT node_port integer,
+												OUT success boolean, OUT result text )
+	RETURNS SETOF record
+	LANGUAGE C STABLE STRICT
+	AS 'citus.so', $$master_run_on_worker$$;
+
+CREATE OR REPLACE FUNCTION citus_run_on_all_workers(command text,
+													parallel bool default true,
+													OUT nodename text,
+													OUT nodeport int,
+													OUT success bool,
+													OUT result text)
+	RETURNS SETOF record
+	LANGUAGE plpgsql
+	AS $function$
+DECLARE
+	workers text[];
+	ports int[];
+	commands text[];
+BEGIN
+	WITH citus_workers AS (
+		SELECT * FROM master_get_active_worker_nodes() ORDER BY node_name, node_port)
+	SELECT INTO workers, ports, commands
+		array_agg(node_name), array_agg(node_port), array_agg(command)
+	FROM citus_workers;
+
```

Comment:
I think we can live without it at this time. We are letting users to perform very unsafe operations at this version.